### DISCUSSION TITLE
$patch across multiple stores?

### DISCUSSION THREAD

**Discussion by kyrel** [vuejs/pinia#1501] [Help and Questions] [answered]
With $patch'es being atomic in terms on $subscribe'rs notified only once for each patch, can I have an atomic update that spans several stores? Or I need to merge stores that can possibly be updated atomically into one store?
Thanks in advance :)

**Comment by posva** [comment, marked as answer]
No, because each store has its own subscriber anyway so each callback will trigger separetely
